packages/pwa-storefront-mcp/src/server/server.test.js

```diff
@@ -4,7 +4,13 @@
  * SPDX-License-Identifier: BSD-3-Clause
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
-import {spawn} from 'child_process'
+// import {spawn} from 'child_process'
+import {spawn} from 'cross-spawn'
+import path from 'path'
+
+const BABEL_NODE_PATH = path.resolve(
+    './node_modules/.bin/babel-node' + (process.platform === 'win32' ? '.cmd' : '')
+)
```

Comment:
We'll run the server with babel-node for now so it doesn't complain about the import syntax of the source files. but in the future we migrate to ESM, maybe add out own test setup we can revert this.